packages/blockchain-extension/extension/commands/addEnvironmentCommand.ts

```diff
@@ -136,6 +182,26 @@ export async function addEnvironment(): Promise<void> {
         }
 
         // Create environment
+        fabricEnvironmentEntry.name = environmentName;
+
+        if (createMethod === UserInputUtil.ADD_ENVIRONMENT_FROM_TEMPLATE) {
+
+            let numberOfOrgs: number;
```

Comment:
if you set the number of orgs as the data property when you choose which template you want you don't need this bit